redi-connect-front/src/services/cached-get.ts

```diff
@@ -0,0 +1,82 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios'
```

Comment:
Doesn't really matter, should work with either, but I saw that we're already using axios. This will help us cache get request if we want to go full offline support.